src/components/MyForums.js

```diff
@@ -151,40 +202,53 @@ const MyForums = () => {
     }*/
 
     return (
-        <div id="body">
-
-            <NavigationBar></NavigationBar>
-
-            <Box sx={{ position: 'absolute', top: 110, left: '50%', transform: 'translate(-50%, -50%)' }}>
-                <Typography
-                    variant="h4"
-                    gutterBottom
-                    component="div">
-                    My Forums
-                </Typography>
-            </Box>
-
-            <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
-            {forums.map((event) => (
-                <Card style={{ width: '800px', marginBottom: '20px' }} key={event.id}>
-                    <CardContent>
-                        <Typography variant="h5" component="div">
-                            {event.forumTitle}<br />
-                        </Typography>
-                        <Typography sx={{ mb: 1.5 }} color="text.secondary">
-                            Posted on {new Date(event.dateTime).toLocaleDateString()}<br />
-                        </Typography>
-                        <Typography sx={{ mb: 1.5 }} color="text.secondary">
-                            <br />{event.description}<br />
-                        </Typography>
-                    </CardContent>
-                </Card>
-            ))}
-            </Box>
+        <div>
+            <div id="body">
+
+                <NavigationBar></NavigationBar>
+
+                <Box sx={{ position: 'absolute', top: 110, left: '50%', transform: 'translate(-50%, -50%)' }}>
+                    <Typography
+                        variant="h4"
+                        gutterBottom
+                        component="div">
+                        My Forums
+                    </Typography>
+                </Box>
+
+                <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
+                    {forums.map((forum) => (
+                        <Card style={{ width: '800px', marginBottom: '20px' }} key={forum.id}>
+                            <CardContent>
+                                <Typography variant="h5" component="div">
+                                    {forum.forumTitle}<br />
+                                </Typography>
+                                <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                    Posted on {new Date(forum.dateTime).toLocaleDateString()}<br />
+                                </Typography>
+                                <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                    <br />{forum.description}<br />
+                                </Typography>
+                                <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                    <br />{forum.status}<br />
```

Comment:
why do you have line breaks? If it's not needed, remove the `<br />` components. This will keep it consistent with the "Forums" page.